hugegraph-api/src/main/java/org/apache/hugegraph/api/arthas/ArthasAPI.java

```diff
@@ -35,6 +37,7 @@
 
 @Path("arthas")
 @Singleton
+@Tag(name = "ArthasAPI")
```

Comment:
could also check other API like **metrics/cypher** if they also missed swagger? And show the current effect pic

Reply:
![image](https://github.com/apache/incubator-hugegraph/assets/48077841/edcf2481-d0b4-4dca-9869-5d3348e3c70f)
yes，i will update other